katz/src/main/kotlin/katz/data/Kleisli.kt

```diff
@@ -0,0 +1,18 @@
+package katz.data
+
+import katz.Functor
+import katz.HK
+import katz.Monad
+
+class Kleisli<F, D, out A>(val run: (D) -> HK<F, A>) {
```

Comment:
Looks good! we should add `pure`, `ask`, `flatten` and any other combinators that may be useful. In addition we need a monad instance for this.